array-methods-task7.js

```diff
@@ -12,5 +12,14 @@ let mary = { name: "Mary", age: 28 };
 let users = [john, pete, mary];
 
 // Own code
-let names = [john.name, pete.name, mary.name].join(", ");
-// console.log(names); // John, Pete, Mary
+// Not generic enough
+// let names = [john.name, pete.name, mary.name].join(", ");
+function arrayOfNames(array) {
+  for (i = 0; i < array.length; i++) {
+    let newArray = array[i].name;
+    newArray = newArray.split(",");
```

Comment:
What are you trying to achieve with this `split`? There is no string in this exercise which require this operation